02/app.js

```diff
@@ -0,0 +1,25 @@
+
+
+const eventMouseOut = function(){
+   
+    const text = 'Mouse has left div';
+    this.innerText=text;
+    console.log(text);
+}
+
+const getMousePos = function(e){
+    const text = `Mouse position x = ${e.pageX}, y = ${e.pageY}`
+    this.innerText=text;
+    console.log(text);
+}
```

Comment:
Osobiście wolę takie funkcje napisać nie jako wyrażenie, ale normalną funkcję.
Wtedy mogę napisać je na końcu, a na początku to co jest najistotniejsze - czyli logikę programu.
Czyli odwróciłbym kolejność całego kodu :)